Generated project does not compile

#### Issue details
The project generation tool -  https://libgdx.badlogicgames.com/nightlies/dist/gdx-setup.jar - doesn't build the project that compiles out of the box.

#### Reproduction steps/code

1. Install Java 8
2. Install Android SDK
3. `cd <path-to-android-sdk>`
4. `sdkmanager "platforms;android-28" "platform-tools" "build-tools;28.0.3"`
5. Run: `grawdlew clean build connectedCheck` on a freshly created project using the above referenced project generation tool.


#### Version of LibGDX and/or relevant dependencies
```
gdxVersion = '1.9.9'
roboVMVersion = '2.3.5'
box2DLightsVersion = '1.4'
ashleyVersion = '1.7.0'
aiVersion = '1.8.0'
```

#### Stacktrace
```java
> Configure project : 
WARNING: Configuration 'compile' is obsolete and has been replaced with 'implementation' and 'api'.
It will be removed at the end of 2018. For more information see: http://d.android.com/r/tools/update-dependency-configurations.html

> Task :core:compileJava 
warning: [options] bootstrap class path not set in conjunction with -source 1.6
1 warning


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':android:lint'.
> Could not resolve all files for configuration ':android:lintClassPath'.
   > Could not find org.jetbrains.trove4j:trove4j:20160824.
     Searched in the following locations:
         file:/<path-to-android-sdk>/extras/m2repository/org/jetbrains/trove4j/trove4j/20160824/trove4j-20160824.pom
         file:/<path-to-android-sdk>/extras/m2repository/org/jetbrains/trove4j/trove4j/20160824/trove4j-20160824.jar
         file:/<path-to-android-sdk>/extras/google/m2repository/org/jetbrains/trove4j/trove4j/20160824/trove4j-20160824.pom
         file:/<path-to-android-sdk>/extras/google/m2repository/org/jetbrains/trove4j/trove4j/20160824/trove4j-20160824.jar
         file:/<path-to-android-sdk>/extras/android/m2repository/org/jetbrains/trove4j/trove4j/20160824/trove4j-20160824.pom
         file:/<path-to-android-sdk>/extras/android/m2repository/org/jetbrains/trove4j/trove4j/20160824/trove4j-20160824.jar
         file:/<user-home-dir>/.m2/repository/org/jetbrains/trove4j/trove4j/20160824/trove4j-20160824.pom
         file:/<user-home-dir>/.m2/repository/org/jetbrains/trove4j/trove4j/20160824/trove4j-20160824.jar
         https://repo.maven.apache.org/maven2/org/jetbrains/trove4j/trove4j/20160824/trove4j-20160824.pom
         https://repo.maven.apache.org/maven2/org/jetbrains/trove4j/trove4j/20160824/trove4j-20160824.jar
         https://dl.google.com/dl/android/maven2/org/jetbrains/trove4j/trove4j/20160824/trove4j-20160824.pom
         https://dl.google.com/dl/android/maven2/org/jetbrains/trove4j/trove4j/20160824/trove4j-20160824.jar
         https://oss.sonatype.org/content/repositories/snapshots/org/jetbrains/trove4j/trove4j/20160824/trove4j-20160824.pom
         https://oss.sonatype.org/content/repositories/snapshots/org/jetbrains/trove4j/trove4j/20160824/trove4j-20160824.jar
         https://oss.sonatype.org/content/repositories/releases/org/jetbrains/trove4j/trove4j/20160824/trove4j-20160824.pom
         https://oss.sonatype.org/content/repositories/releases/org/jetbrains/trove4j/trove4j/20160824/trove4j-20160824.jar
     Required by:
         project :android > com.android.tools.lint:lint-gradle:26.2.0 > com.android.tools.external.com-intellij:intellij-core:26.2.0

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
See https://docs.gradle.org/4.6/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 11s
61 actionable tasks: 55 executed, 6 up-to-date
```

#### Please select the affected platforms
- [x] Android
- [x] iOS (robovm)
- [ ] iOS (MOE)
- [x] HTML/GWT
- [x] Windows
- [x] Linux
- [x] MacOS

(I did not tick `Ios-moe` on the project generation form)

--- a/extensions/gdx-setup/src/com/badlogic/gdx/setup/BuildScriptHelper.java
+++ b/extensions/gdx-setup/src/com/badlogic/gdx/setup/BuildScriptHelper.java
@@ -77,6 +77,7 @@ public class BuildScriptHelper {
 		write(wr, "repositories {");
 		write(wr, DependencyBank.mavenLocal);
 		write(wr, DependencyBank.mavenCentral);
+		write(wr, DependencyBank.jCenter);
 		write(wr, DependencyBank.google);
 		write(wr, "maven { url \"" + DependencyBank.libGDXSnapshotsUrl + "\" }");
 		write(wr, "maven { url \"" + DependencyBank.libGDXReleaseUrl + "\" }");

